server/pulp/server/webservices/views/repositories.py

```diff
@@ -121,6 +121,35 @@ def _process_repos(repos, details, importers, distributors):
     return repos
 
 
+def _validate_repo_importer_existance(repo_id, importer_id):
```

Comment:
With a name like this, I do not expect it to return the importer. Perhaps this should be `_get_importer` or something similar. I think it is reasonable that this function could both retrieve and validate.